.github/workflows/ci-sharktank.yml

```diff
@@ -98,7 +97,6 @@ jobs:
 
```

Comment:
We could add missed changes from [here](https://github.com/nod-ai/shark-ai/pull/639#discussion_r1868485174).

Reply:
Keeping PRs small and focused. Sent https://github.com/nod-ai/shark-ai/pull/642 for that.